website/client/src/components/tasks/task.vue

```diff
@@ -995,18 +992,19 @@ export default {
       this.scoreChecklistItem({ taskId: this.task._id, itemId: item.id });
     },
     calculateTimeTillDue () {
-      const endOfToday = moment().endOf('day');
-      const endOfDueDate = moment(this.task.date).endOf('day');
+      const dayEnd = m => m.endOf('day').add(this.user.preferences.dayStart, 'hours');
+
+      const endOfToday = dayEnd(moment());
```

Comment:
it looks like this value is shifted one day in the future, (right now it's 11am on the 25th and my CDS is set at 5pm) and if printed it says `Sat Sep 26 2020 16:59:59 GMT+0200`

Reply:
Looking at other places that handle time calculations with CDS, I think that if `currentHourOfTheDay < CDS` you should subtract 1 day and then add the dayStart preference, otherwise `dayEnd + CDS` will go to tomorrow.

Or if ``currentHourOfTheDay < CDS` use `startOf('day')` instead of `endOf('day')`

Reply:
Ah yeah, that makes sense.  Thanks a lot!  Gives me another opportunity to at least attempt a unit test 😉.

Reply:
Nah, that was probably just rebasing on `develop`. As mentioned somewhere else, my laptop broke before I could push the unit test I wrote for this story, so will look into re-making that somewhere this week.

Reply:
looking good now!